tests/_files/Report/XML/CoverageForBankAccount/BankAccount.php.xml

```diff
@@ -35,7 +35,7 @@
         <covered by="BankAccountTest::testDepositWithdrawMoney"/>
       </line>
     </coverage>
-    <source>
+    <source xmlns="https://schema.phpunit.de/coverage/1.0">
```

Comment:
@staabm @theseer Are these changes intentional?

Reply:
I think its not a change from a semantic point of view.

before this PR we imported one document into another which lead to the involved php extensions normalizing this namespace away. technically it is not necessary because its the same namespace as in the root phpunit element, but as this normalization no longer happens this `xmlns="https://schema.phpunit.de/coverage/1.0"` is showing up now.